.editorconfig

```diff
@@ -7,6 +7,7 @@ indent_size = 4
 charset = utf-8
```

Comment:
So is it possible to test out policies in a forked branch?  (I hate that actions and build scripts are difficult to verify pre-merge.)

Reply:
Unfortunately not. I was able to test out the GH action declared in `.github/workflows/run-formatter.yml` before opening the PR (#11910 was opened automatically when I enabled branch protection on the `jeskew/automated-formatting` branch; a prior run pushed directly to `jeskew/automated-formatting` before branch protection was enabled).

Github policies, however, will by design only run once they are committed to the repo's default branch. I reached out to the team that maintains it to ask what the validation message in https://github.com/Azure/bicep/pull/11911#issuecomment-1730313140 means but have not heard back, but the GitOps.YmlValidation workflow is basically the only assurance we get that a policy will work as intended (or at all).